lib/puppetlabs_spec_helper/module_spec_helper.rb

```diff
@@ -19,6 +20,7 @@ def verify_contents(subject, title, expected_lines)
 module_path = [module_path, env_module_path].join(File::PATH_SEPARATOR) if env_module_path
 
 RSpec.configure do |c|
+  c.environmentpath = spec_path
```

Comment:
As it is just setting rspec settings, I'm not worried about it affecting puppet. It's up to rspec-puppet to use the settings or ignore the setting based on the puppet version.